pkg/server/http/svc_test.go

```diff
@@ -0,0 +1,761 @@
+}
+
+type DeleteUserRespone struct {
+	Meta Meta `json:"meta" xml:"meta"`
+	Data struct {
+	} `json:"data" xml:"data"`
+}
+
+func assertResponseMeta(t *testing.T, expected, actual Meta) {
+	assert.Equal(t, expected.Status, actual.Status, "The status of response doesn't matches")
+	assert.Equal(t, expected.StatusCode, actual.StatusCode, "The Status code of response doesn't matches")
+	assert.Equal(t, expected.Message, actual.Message, "The Message of response doesn't matches")
+}
+
+//func assertUserSame(t *testing.T, expected, actual User) {
+//assert.Equal(t, expected.ID, actual.ID, "UserId doesn't match for user %v", expected.Username)
+//assert.Equal(t, expected.Username, actual.Username, "Username doesn't match for user %v", expected.Username)
+//assert.Equal(t, expected.Email, actual.Email, "email doesn't match for user %v", expected.Username)
+
+//assert.Equal(t, expected.Enabled, actual.Enabled, "enabled doesn't match for user %v", expected.Username)
+//assert.Equal(t, expected.Quota, actual.Quota, "Quota match for user %v", expected.Username)
+//assert.Equal(t, expected.UIDNumber, actual.UIDNumber, "UidNumber doesn't match for user %v", expected.Username)
+//assert.Equal(t, expected.GIDNumber, actual.GIDNumber, "GIDNumber doesn't match for user %v", expected.Username)
+//assert.Equal(t, expected.Displayname, actual.Displayname, "displayname doesn't match for user %v", expected.Username)
+//}
+
+const dataPath = "./accounts-store"
+
+func deleteAccount(t *testing.T, id string) (*empty.Empty, error) {
+	client := service.Client()
+	cl := accountsProto.NewAccountsService("com.owncloud.api.accounts", client)
+
+	req := &accountsProto.DeleteAccountRequest{Id: id}
+	res, err := cl.DeleteAccount(context.Background(), req)
+	return res, err
+}
+
+func init() {
+	service = grpc.NewService(
+		grpc.Namespace("com.owncloud.api"),
+		grpc.Name("accounts"),
+		grpc.Address("localhost:9180"),
+	)
+
+	cfg := accountsCfg.New()
+	cfg.Server.AccountsDataPath = dataPath
+	var hdlr *accountsSvc.Service
+	var err error
+
+	if hdlr, err = accountsSvc.New(
+		accountsSvc.Logger(accountsCmd.NewLogger(cfg)),
+		accountsSvc.Config(cfg)); err != nil {
+		log.Fatalf("Could not create new service")
+	}
+
+	err = accountsProto.RegisterAccountsServiceHandler(service.Server(), hdlr)
+	if err != nil {
+		log.Fatal("could not register the Accounts handler")
+	}
+	err = accountsProto.RegisterGroupsServiceHandler(service.Server(), hdlr)
+	if err != nil {
+		log.Fatal("could not register the Groups handler")
+	}
+
+	err = service.Server().Start()
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func cleanUp(t *testing.T) {
+	datastore := filepath.Join(dataPath, "accounts")
+
+	files, err := ioutil.ReadDir(datastore)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, f := range files {
+		found := false
+		for _, defUser := range DefaultUsers {
+			if f.Name() == defUser {
+				found = true
+				break
+			}
+		}
+
+		if !found {
+			deleteAccount(t, f.Name())
```

Comment:
catch err from deleteAccount
```suggestion
			_, err = deleteAccount(t, f.Name())
			if err != nil {
			    log.Fatal(err)
			}
```